src/StandbyPool/StandbyPool.Autorest/README.md

```diff
@@ -129,6 +129,13 @@ directive:
     set:
       parameter-name: Version
 
+  - where:
+      verb: New|Update
+      subject: StandbyContainerGroupPool
+      parameter-name: Zone
+    set:
```

Comment:
This is not allowed. In PowerShell, we should use singular noun for parameter name.